pyproject.toml

```diff
@@ -1,7 +1,84 @@
 [build-system]
-requires = ["setuptools", "wheel"]
+requires = ["setuptools"]
```

Comment:
setuptools automatically installs wheel if needed. Further info is in the first note in this section: https://setuptools.pypa.io/en/latest/userguide/quickstart.html#basic-use